src/Chromatic/Sniffs/Commenting/TodoTicketReferenceCommentSniff.php

```diff
@@ -52,7 +52,7 @@ public function process(File $phpcsFile, $stackPtr)
         $tokens     = $phpcsFile->getTokens();
         $expression = '/@todo(?::| )(?:(?!((http|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])?)).)*$/mi';
         if ((bool) preg_match($expression, $tokens[$stackPtr]['content']) === true) {
-            $phpcsFile->addError('TODO comments require a ticket reference', $stackPtr, 'TodoTicketReference');
+            $phpcsFile->addError('@todo comments require a ticket reference', $stackPtr, 'TodoTicketReference');
```

Comment:
@markdorison did we intentionally use four spaces instead of two for indentation?

Reply:
I followed the example set by the [coder module](https://git.drupalcode.org/project/coder). While they implement the Drupal coding standard, they [do not _follow_](https://git.drupalcode.org/project/coder/-/blob/8.x-3.x/phpunit.xml.dist) the Drupal coding standard

Reply:
Terribly ironic. 😂

Reply:
Well, it's not a Drupal module, so I can see that it doesn't really make sense to follow Drupal's coding standards.

Reply:
Ah, right. Coder is agnostic. It's just the DrupalPractice bit that is Drupal.